client/src/services/getCameraData.js

```diff
@@ -1,6 +1,9 @@
 // @flow
-export default function() {
-  const position: Array<number> = window.mathbox.three.camera.position.toArray()
-  const lookAt: Array<number> = window.mathbox.three.controls.center.toArray()
-  return { position, lookAt }
+
+import { mathbox } from "containers/MathBoxScene/components/MathBoxScene.js";
+
+export default function () {
+  const position: Array<number> = mathbox.three.camera.position.toArray();
+  const lookAt: Array<number> = mathbox.three.controls.target.toArray();
```

Comment:
this change from `center` to `target` happened in `OrbitControls` a few versions ago.